pkg/controller/add_kogitotrusty.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019 Red Hat, Inc. and/or its affiliates
```

Comment:
This file should be generated by the operator sdk

Reply:
Done :+1: